src/components/interface/dialog/DialogHeader.jsx

```diff
@@ -32,6 +33,7 @@ DialogHeader.propTypes = {
   classes: PropTypes.shape().isRequired,
   collectionName: PropTypes.string.isRequired,
   edit: PropTypes.bool,
+  handleDup: PropTypes.func,
```

Comment:
alphabetize please

Reply:
@wrjren This one actually isn't fully alphabetized, can we ensure the entire object is alphabetized. docId should be above edit. Not your fault on this one :)